.vscode/spellright.dict

```diff
@@ -0,0 +1,3 @@
+github
```

Comment:
If this is here, the  **Spell Right** has to be added to the recommended plugins.